src/components/fetch/useScrollFetch.js

```diff
@@ -2,27 +2,27 @@ import { useState, useEffect } from 'react';
 
 const LIMIT = 10;
 
-const useScrollFetch = (url, page) => {
+const useScrollFetch = (url, page, query) => {
   const [pins, setPins] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(false);
 
   useEffect(() => {
-    const query = `?limit=${LIMIT}&offset=${LIMIT * page}`;
-
-    fetch(url + query)
+    const pagnation = `?limit=${LIMIT}&offset=${LIMIT * page}`;
+    if (query === undefined) query = '';
+    fetch(url + pagnation + query)
       .then(res => res.json())
```

Comment:
query의 boolean값에 따라 작성해줄수도 있겠네요!
또는 fetch 인자에서 로직을 작성할 수도 있습니다.
```suggestion
    if (!query) query = '';
```